tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks file for ansible-role-bes-docker
+
+- include: "tasks/oah-install.yml"
+  when: oah_command == "install"
+
+- include: "tasks/oah-remove.yml"
+  when: oah_command == "uninstall"
```

Comment:
change in command

Reply:
Changed the command from uninstall -> remove